packages/encrypt-upload-client/src/config/service.js

```diff
@@ -5,7 +5,7 @@ import { gatewayServiceConnection } from '@storacha/client/service'
 
 const storachaServiceURL = 'https://up.web3.storage'
 const storachaPrincipalDID = 'did:web:web3.storage'
-
+//TODO: import from service
```

Comment:
what are we supposed to be doing here that we haven't done?

Reply:
I will improve that comment. It needs to be done after the repo unification task is completed because the did web that comes from the w3up-client/service package is `did:web:up.storacha.network`.